scheduling/task_run.go

```diff
@@ -44,49 +46,74 @@ func (trmk TaskRunMetadataKey) String() string {
 }
 
 type TaskRunID ffsync.OrderedId
-type Status string
+
+func NewTaskRunIdFromString(id string) TaskRunID {
+	var orderedID ffsync.Uint64OrderedId
+	orderedID.FromString(id)
```

Comment:
need to handle the `err` case otherwise will return a value of 0 which could cause logical problems.